### PR TITLE
Added descriptions to enums

### PR DESCRIPTION
**Pull request by rocketsofthsv**
@mairaw

### CODE REVIEW

File: xml/Microsoft.Vsa/VsaError.xml [hunk 1]
```diff
@@ -107,7 +107,7 @@
       </ReturnValue>
       <MemberValue>-2146226173</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The application base is invalid</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The application base is invalid.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 2]
```diff
@@ -128,7 +128,7 @@
       </ReturnValue>
       <MemberValue>-2146226172</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The assembly is excpected</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The assembly is expected.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 3]
```diff
@@ -149,7 +149,7 @@
       </ReturnValue>
       <MemberValue>-2146226171</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The assembly name is invalid</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The assembly name is invalid.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 4]
```diff
@@ -212,7 +212,7 @@
       </ReturnValue>
       <MemberValue>-2146226169</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The cached assembly is invalid</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The cached assembly is invalid.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 5]
```diff
@@ -233,7 +233,7 @@
       </ReturnValue>
       <MemberValue>-2146226168</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The call back was unexpected</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The callback was unexpected.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 6]
```diff
@@ -317,7 +317,7 @@
       </ReturnValue>
       <MemberValue>-2146226114</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The debugger was not started.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The debuggee was not started.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 7]
```diff
@@ -422,7 +422,7 @@
       </ReturnValue>
       <MemberValue>-2146226161</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The eninge cannot close.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The engine cannot close.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 8]
```diff
@@ -527,7 +527,7 @@
       </ReturnValue>
       <MemberValue>-2146226156</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The engine is not in use.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The engine name is in use.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 9]
```diff
@@ -548,7 +548,7 @@
       </ReturnValue>
       <MemberValue>-2146226113</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The engine is name is invalid.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The engine name is invalid.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 10]
```diff
@@ -632,7 +632,7 @@
       </ReturnValue>
       <MemberValue>-2146226154</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The engine not initialized.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The engine is not initialized.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 11]
```diff
@@ -737,7 +737,7 @@
       </ReturnValue>
       <MemberValue>-2146226149</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Event source name is invalid.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The event source name is invalid.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 12]
```diff
@@ -758,7 +758,7 @@
       </ReturnValue>
       <MemberValue>-2146226148</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Event source not found.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The event source was not found.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 13]
```diff
@@ -779,7 +779,7 @@
       </ReturnValue>
       <MemberValue>-2146226147</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Event source type is invalid.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The event source type is invalid.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 14]
```diff
@@ -926,7 +926,7 @@
       </ReturnValue>
       <MemberValue>-2146226142</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The item could not be removed.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The item cannot be removed.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 15]
```diff
@@ -1031,7 +1031,7 @@
       </ReturnValue>
       <MemberValue>-2146226138</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The item name was not found.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The item was not found.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 16]
```diff
@@ -1052,7 +1052,7 @@
       </ReturnValue>
       <MemberValue>-2146226137</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The item type not supported.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The item type is not supported.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 17]
```diff
@@ -1283,7 +1283,7 @@
       </ReturnValue>
       <MemberValue>-2146226105</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The proccess name is in use.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The process name is in use.</summary>
```

File: xml/Microsoft.Vsa/VsaError.xml [hunk 18]
```diff
@@ -1388,7 +1388,7 @@
       </ReturnValue>
       <MemberValue>-2146226128</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The root moniker is in use.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaError.xml]:
```suggestion
        <summary>The root moniker is invalid.</summary>
```

File: xml/Microsoft.Vsa/VsaItemType.xml [hunk 19]
```diff
@@ -44,7 +44,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The VSA item type is global.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaItemType.xml]:
```suggestion
        <summary>The item type is AppGlobal.</summary>
```

File: xml/Microsoft.Vsa/VsaItemType.xml [hunk 20]
```diff
@@ -65,7 +65,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The VSA item type is code.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaItemType.xml]:
```suggestion
        <summary>The item type is Code.</summary>
```

File: xml/Microsoft.Vsa/VsaItemType.xml [hunk 21]
```diff
@@ -86,7 +86,7 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The VSA item type is a reference.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaItemType.xml]:
```suggestion
        <summary>The item type is Reference.</summary>
```

**Comment by mairaw** [comment]
You can batch all my suggestions into a single commit via the Files changes tab (https://github.com/dotnet/dotnet-api-docs/pull/3573/files).

**Comment by rocketsofthsv** [comment]
@mairaw committed your suggestions

**Comment by mairaw** [comment]
I still see some not commited. Go to https://github.com/dotnet/dotnet-api-docs/pull/3573/files to see them all and add them in batch.

**Pull request merged**